CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 0.5.2 (Next)
 ============
 
+* [#215](https://github.com/ruby-grape/grape-entity/pull/217): `#delegate_attribute` should not delegate to methods included with `Kernel`.
```

Comment:
One last nitpick and then I stop. See the format of these below, your name should be at the end, so `- [@...]().`. Thanks.